x-pack/plugins/alerting/server/task_runner/log_alerts.ts

```diff
@@ -130,7 +128,7 @@ export function logAlerts<
         group: actionGroup,
         message,
         state,
-        flapping: false,
+        flapping: activeAlerts[id].getFlapping(),
```

Comment:
**question:** Do we need to do the same within the `for (const id of newAlertIds) {` code above?

Reply:
I don't think so, the new alerts will always be false

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/144415/commits/bd748cd1fa3fdbcb3cdd8689b593a9088afd6756